sql/group.go

```diff
@@ -181,6 +183,36 @@ func (n *groupNode) wrap(plan planNode) planNode {
 	return n
 }
 
+// desiredAggregateOrdering computes the desired output ordering from the
+// scan. It looks for an output column index containing a simle MIN/MAX
+// aggregation. If zero or multiple MIN/MAX aggregations are requested then no
+// ordering will be requested. A negative index indicates a MAX aggregation was
+// requested for the output column.
+func desiredAggregateOrdering(funcs []*aggregateFunc) []int {
```

Comment:
Test added. It's pretty simple because any more complex case reduces to the cases given.